src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-fri calendar--monts-length-30">
```

Comment:
There is a typo in the class name 'calendar--monts-length-30', it should be 'calendar--month-length-30'